lib/api/src/js/externals/memory.rs

```diff
@@ -243,6 +243,22 @@ impl Memory {
         self.view.clone()
     }
 
+    /// A theoretical alias to `Self::view::<u8>` but it returns a `js::Uint8Array` in this case.
+    ///
+    /// This code is going to be refactored. Use it as your own risk.
+    #[doc(hidden)]
+    pub fn uint8view_with_byte_offset_and_length(
```

Comment:
This function doesn't seem to actually be called anywhere? Did this get lost with the merge?